app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,53 @@
+class Api::UsersController < ApplicationController
+    before_action :set_user, only: [:show, :update, :destroy]
+    before_action :require_logged_in, only: [:show, :update, :destroy]
```

Comment:
I believe you can combine these: `before_action :set_user, :require_logged_in, only: [:show, :update, :destroy]
`